schemas/com.shopify/update-product/jsonschema/1-0-0

```diff
@@ -0,0 +1,272 @@
+          },
+          "values" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
+          },
+          "position" : {
+            "type" : "integer",
+            "maximum" : 32767,
+            "minimum" : 0
+          }
+        },
+        "additionalProperties" : false
+      }
+    },
+    "id" : {
+      "type" : "integer",
+      "maximum" : 9223372036854775807,
+      "minimum" : 0
+    },
+    "created_at" : {
+      "type" : "string",
+      "format" : "date-time"
+    },
+    "title" : {
+      "type" : "string"
+    },
+    "template_suffix" : {
+      "type" : "null"
+    },
+    "vendor" : {
+      "type" : "string"
+    },
+    "updated_at" : {
+      "type" : "string",
+      "format" : "date-time"
+    },
+    "images" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "variant_ids" : {
+            "type" : "array",
+            "items" : {
+              "type" : "integer",
+              "maximum" : 9223372036854775807,
+              "minimum" : 0
+            }
+          },
+          "id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "src" : {
+            "type" : "string",
+            "format" : "uri"
+          },
+          "product_id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "position" : {
+            "type" : "integer",
+            "maximum" : 32767,
+            "minimum" : 0
+          },
+          "created_at" : {
+            "type" : "string",
+            "format" : "date-time"
+          },
+          "updated_at" : {
+            "type" : "string",
+            "format" : "date-time"
+          }
+        },
+        "additionalProperties" : false
+      }
+    },
+    "published_at" : {
+      "type" : [ "string", "null" ],
+      "format" : "date-time"
+    },
+    "variants" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "weight" : {
+            "type" : "number",
+            "minimum" : 0
+          },
+          "inventory_policy" : {
+            "type" : "string",
+            "pattern" : "^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$"
+          },
+          "sku" : {
+            "type" : "string"
+          },
+          "barcode" : {
+            "type" : "string"
+          },
+          "image_id" : {
+            "type" : [ "integer", "null" ],
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "option1" : {
+            "type" : "string"
+          },
+          "price" : {
+            "type" : "string"
+          },
+          "weight_unit" : {
+            "type" : "string",
+            "enum" : [ "g", "kg", "oz", "lb" ]
```

Comment:
Good candidate for `enum`, but do we really have all items here?

Reply:
Not sure here, but i remember going through the shopify API docs to see, if they have a fixed set of possible values. If yes, i created an enum set.

Reply:
> weight_unit: Unit that the product variant's weight is measured in. Valid values: g, kg, oz, lb.